addons/mass_mailing/static/src/css/basic_theme_readonly.css

```diff
@@ -5,3 +5,10 @@
 #wrapwrap .o_layout.o_basic_theme {
     font-family: -apple-system, "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
 }
+
+/*
+Make sure that the mail body takes all available space
+ */
+body.o_readonly {
```

Comment:
Seems this should be namespaced ? Doesn't that have side effects ?

Reply:
Changed to something more specific, however, since this css is only applied on the content of the body_html iframe, I don't think that we can have side effects with this.